src/_includes/docs/add-to-app/ios-project/embed-frameworks.md

```diff
@@ -0,0 +1,117 @@
+### Embed frameworks in Xcode {:#method-b .no_toc}
+
+#### Approach {:#method-b-approach}
+
+In this second method, edit your existing Xcode project,
+generate the necessary frameworks, and embed them in your app. You need to create three iOS frameworks: for the Flutter engine, for your compiled Dart code, and for all Flutter plugins in Xcode. Embed these frameworks and update your existing application's build settings.
+
+Every time you change code in your Flutter module,
+run `flutter build ios-framework`.
+
+#### Requirements {:#method-b-reqs}
+
+No additional software or hardware requirements are needed for this method.
+Use this method in the following use cases:
+
+* Members of your team can't install the Flutter SDK and CocoaPods
+* You don't want to use CocoaPods as a dependency manager in existing iOS apps
+
+#### Limitations {:#method-b-limits}
+
+{% render docs/add-to-app/ios-project/limits-common-deps.md %}
+
+#### Example project structure {:#method-b-structure}
+
+The following example assumes that you want to generate the
+frameworks to `/path/to/MyApp/Flutter/`.
+
+```console
+$ flutter build ios-framework --output=/path/to/MyApp/Flutter/
+```
+
+The resulting project structure should resemble this directory tree.
+
+```plaintext
+/path/to/MyApp/
+└── Flutter/
+    ├── Debug/
+    │   ├── Flutter.xcframework
+    │   ├── App.xcframework
+    │   ├── FlutterPluginRegistrant.xcframework (only if you have plugins with iOS platform code)
+    │   └── example_plugin.xcframework (each plugin is a separate framework)
+    ├── Profile/
+    │   ├── Flutter.xcframework
+    │   ├── App.xcframework
+    │   ├── FlutterPluginRegistrant.xcframework
+    │   └── example_plugin.xcframework
+    └── Release/
+        ├── Flutter.xcframework
+        ├── App.xcframework
+        ├── FlutterPluginRegistrant.xcframework
+        └── example_plugin.xcframework
+```
+
+:::warning
+Always put `Flutter.xcframework` and `App.xcframework` from the same directory.
+Mixing `.xcframework` imports from different directories
+(like `Profile/Flutter.xcframework` with `Debug/App.xcframework`)
+causes runtime crashes.
+:::
+
+#### Procedures
+
+You can either link or embed the generated frameworks
+into your existing app in Xcode.
+Use the method that is best for your project.
+
+##### Link on the frameworks
+
+Continuing with this example,
+you can drag the frameworks from
+the `/path/to/MyApp/Flutter/Release/` directory in the Finder
+to your target's
+**Build Settings > Build Phases > Link Binary With Libraries** settings.
```

Comment:
Could we add a screenshot for this? I think I had provided one before, let me know if you need a new screenshot :)